client/src/routes/routes.ts

```diff
@@ -115,7 +115,7 @@ const routes: RouteMap = {
   User: {
     component: User,
     roles: ['OWNER', 'ADMIN'],
-    path: '/users/:userId',
+    path: '/profile',
```

Comment:
i think we'll want to show other users', not just the current user, profile page. so i think this isn't the right fix
we should instead fix the link to point to /users/[current_user.id]

Reply:
So, the `user-id` path param determines the public user profile to fetch?

Reply:
that's how i imagined it/the most common pattern i've seen. if you have another you prefer then that's fine with me

Reply:
Sure, so i guess what you had originally is fine

So both urls point to the same react component
logic in there would be: 
Use url param user ID
If doesn't exist, use current user